back-end/bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
+var app = require('../app');
```

Comment:
Was this automatically generated? Wondering why we're using `var` here instead of `const` or `let` 🤔  like in the other files.

If this was made by us, can we change these please?

Reply:
This was auto-genereated by express-generator, and I did not spend any time updating the syntax in it. I can do that, though!

Reply:
Updated!